profiles.md

```diff
@@ -0,0 +1,208 @@
+        "authority": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+        "confidence": 600,
+        "place": 0
+      }],
+    "cris.sourceId": [{
+        "value": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.date.accessioned": [{
+        "value": "2020-07-20T15:31:11Z",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.date.available": [{
+        "value": "2020-07-20T15:31:11Z",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.description.provenance": [{
+        "value": "Made available in DSpace on 2020-07-20T15:31:11Z (GMT). No. of bitstreams: 0",
+        "language": "en",
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.identifier.uri": [{
+        "value": "http://localhost:4000/handle/123456789/511",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "relationship.type": [{
+        "value": "Person",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "inArchive": true,
+  "discoverable": true,
+  "withdrawn": false,
+  "lastModified": "2020-07-20T16:02:19.608+0000",
+  "type": "item"
+}
+```
+It would respond with:
+* 200 OK - Returning the item if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the item
+* 404 Not found - if the item doesn't exist
+
+### EPerson
+**GET /api/cris/profiles/<:eperson-uuid>/eperson**
+
+Returns the eperson owning the profile. The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "uuid": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "name": "msmith@samltest.id",
+  "handle": null,
+  "metadata": {
+    "eperson.firstname": [{
+        "value": "Mortimer",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "eperson.lastname": [{
+        "value": "Smith",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "netid": "msmith@samltest.id",
+  "lastActive": "2019-10-25T08:05:46.530+0000",
+  "canLogIn": true,
+  "email": "msmith@samltest.id",
+  "requireCertificate": false,
+  "selfRegistered": false,
+  "type": "eperson"
+}
+```
+
+It would respond with:
+* 200 OK - Returning the eperson if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the eperson
+* 404 Not found - if the eperson doesn't exist
+
+## Creating a new Profile
+**POST /api/cris/profiles?eperson=<:eperson-uuid>**
```

Comment:
Should there be an option to link up an *existing EPerson* with an *existing Person Entity*?  Currently, it looks like this assumes we are creating a *new* Person Entity to link to the EPerson.  My assumption is that there may be scenarios where the Person Entity _already exists_ and the EPerson just wants to "claim" it as being theirs.

Reply:
Yes, I will add this option to the contract.